### PR TITLE
File added : Bootstrap assignment_1

### PR DESCRIPTION
**Pull request by nileshp1122**

### CODE REVIEW

File: Nilesh/Bootstrap/Assignment1/nileshA1_Q1.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                                <br />
+                                <div class="float-right"><a href="">View all</a></div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-9">
+                        <div class="card" style="box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.2)">
+                            <div class="float-left"><i class="fa fa-user-circle-o" style="font-size: 24px; padding-left: 20px; margin: 23px 0 0 0; color: gray"></i></div>
+                            <div class="card-body" style="float: right; font-size: 13px; margin: -46px 0 0 40px">Announce something to your class</div>
+                            <div class="float-right"><i class="fa fa-retweet" style="float: right; font-size: 14px; color: gray; line-height: 50px; margin: -55px 24px 0 0"></i></div>
```
Comment (Ashwin2812) [inline Nilesh/Bootstrap/Assignment1/nileshA1_Q1.html:41]: Use Correct Bootstrap classes in line 39 , 41

File: Nilesh/Bootstrap/Assignment1/nileshA1_Q1.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                        </div>
+                        <div class="card" style="margin-top: 20px">
+                            <div class="float-left" style="margin: 0 0 0 19px"><i class="fa fa-clipboard" style="font-size: 14px; margin-top: 18px; background-color: rgb(38, 0, 255); border-radius: 50%; padding: 6px; color: white"></i></div>
+                            <div class="card-body" style="float: right; font-size: 13px; margin: -43px 0 0 40px">Pawan Jagtap posted a new material : Bootstrap - Document</div>
+                            <div class="float-right"><i class="fa fa-ellipsis-v" style="float: right; font-size: 14px; color: gray; line-height: 50px; margin: -55px 30px 0 0"></i></div>
```
Comment (Ashwin2812) [inline Nilesh/Bootstrap/Assignment1/nileshA1_Q1.html:46]: Use correct Bootstrap classes in line 44 , 46

File: Nilesh/Bootstrap/Assignment1/nileshA1_Q1.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                        </div>
+                        <div class="card" style="margin-top: 20px">
+                            <div class="float-left" style="margin: 0 0 0 19px"><i class="fa fa-clipboard" style="font-size: 14px; margin: 18px 0 0 0; background-color: rgb(38, 0, 255); border-radius: 50%; padding: 6px; color: white"></i></div>
+                            <div class="card-body" style="float: right; font-size: 13px; margin: -43px 0 0 40px">Pawan Jagtap posted a new assignment : Evaluation 26 May 2021</div>
+                            <div class="float-right"><i class="fa fa-ellipsis-v" style="float: right; font-size: 14px; color: gray; line-height: 50px; margin: -55px 30px 0 0"></i></div>
```
Comment (Ashwin2812) [inline Nilesh/Bootstrap/Assignment1/nileshA1_Q1.html:51]: Use correct Bootstrap classes in line 49 , 51

**Review by Ashwin2812 — CHANGES_REQUESTED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

File: Nilesh/Bootstrap/Assignment1/nileshA1_Q1.html (hunk 1 above)
Comment (nileshp1122) [reply Nilesh/Bootstrap/Assignment1/nileshA1_Q1.html:41]: Read bootstrap float class

File: Nilesh/Bootstrap/Assignment1/nileshA1_Q1.html (hunk 2 above)
Comment (nileshp1122) [reply Nilesh/Bootstrap/Assignment1/nileshA1_Q1.html:46]: Read float class in bootstrap

**Review by Ashwin2812 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**